setup.py

```diff
@@ -37,6 +37,7 @@ def read(filename):
         'Framework :: Django :: 2.0',
         'Framework :: Django :: 2.1',
         'Framework :: Django :: 2.2',
+        'Framework :: Django :: 3.0',
```

Comment:
I requested a Trove classifier, it hasn't been added yet.

https://github.com/pypa/warehouse/issues/6605

Reply:
Should I remove that line for now?

Reply:
You can remove it or comment it out,  either way.